### PR TITLE
Dev sprint organizer post

### PR DESCRIPTION
**Pull request by kojoidrissa**
Changes proposed in this PR:

- Later than I'd hoped, but it's here: things to consider if you want to lead or help lead a sprint

### CODE REVIEW

File: _posts/2022-07-08-dev-sprint-leaders.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+## Why lead/organize a sprint at DCUS 2022?
+There are as many reasons to lead or organize a development sprint as there are projects to work on and potential organizers. Some of the key reasons to consider are:
+- Meet & work with current contributors in person!
+- Test & refine your new user & new contributor docs with live feedback. Better new contributor docs lead to better new issues, bug fixes and PRs for your project.
+- Build/improve a workflow for virtual attendees that your project can continue to use after DjangoCon US (when EVERYONE will be 'virtual').
+- Get help for your favorite project from enthusiastic Pythonistas & Djangonauts, while enjoying beautiful San Diego!
+
+
+## Preparation for Project Maintainers: Avoiding Burnout
+Let's start with this acknowledgement: the vast majority of FLOSS project maintainers are unpaid volunteers using their free time to maintain software that many us depend on. Your work is greatly appreciated by all of us *(even those who don't explicitly say so*). With this in mind, most recommendations here are focused on ways to help maintainers get the most from a Development Sprint in the most scalable, least burdensome way possible.
```
Comment (drewbrew) [inline _posts/2022-07-08-dev-sprint-leaders.md]:
```suggestion
Let's start with this acknowledgement: the vast majority of FLOSS project maintainers are unpaid volunteers using their free time to maintain software that many us depend on. Your work is greatly appreciated by all of us *(even those who don't explicitly say so)*. With this in mind, most recommendations here are focused on ways to help maintainers get the most from a Development Sprint in the most scalable, least burdensome way possible.
```

Just moving the close italics to match the parentheses here

File: _posts/2022-07-08-dev-sprint-leaders.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+A bit of preparation will help you and your chosen project get the most from the Development Sprints, as well as help potential sprinters have the best possible experience working on your project. Instead of the traditional approach of making/marking "beginner friendly" issues, try making sure your "Start Contributing" docs are comprehensive and up to date. If your project doesn't have a CONTRIBUTING.md/rst doc, there are several examples you can look to for help, such as the two different [DRF](https://github.com/encode/django-rest-framework/blob/master/CONTRIBUTING.md) [docs](https://www.django-rest-framework.org/community/contributing/) or the [pytest](https://github.com/pytest-dev/pytest/blob/main/CONTRIBUTING.rst) docs. You can also look at the [Django](https://docs.djangoproject.com/en/dev/internals/contributing/) or [CPython](https://devguide.python.org/) developer docs. Creating or updating these docs prior to the DCUS Dev Sprints will have two main benefits:
```
Comment (drewbrew) [inline _posts/2022-07-08-dev-sprint-leaders.md]:
```suggestion
A bit of preparation will help you and your chosen project get the most from the Development Sprints, as well as help potential sprinters have the best possible experience working on your project. Instead of the traditional approach of making/marking "beginner-friendly" issues, try making sure your "Start Contributing" docs are comprehensive and up to date. If your project doesn't have a CONTRIBUTING.md/rst doc, there are several examples you can look to for help, such as the two different [DRF](https://github.com/encode/django-rest-framework/blob/master/CONTRIBUTING.md) [docs](https://www.django-rest-framework.org/community/contributing/) or the [pytest](https://github.com/pytest-dev/pytest/blob/main/CONTRIBUTING.rst) docs. You can also look at the [Django](https://docs.djangoproject.com/en/dev/internals/contributing/) or [CPython](https://devguide.python.org/) developer docs. Creating or updating these docs prior to the DCUS Dev Sprints will have two main benefits:
```

Just adding a hyphen

File: _posts/2022-07-08-dev-sprint-leaders.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+- You'll be able to get live feedback **on** that documentation from new contributors in person DURING the sprint.
+
+[Mike McQuaid](https://github.com/MikeMcQuaid), the [Homebrew](https://github.com/Homebrew) project leader, has some [useful insights](https://mikemcquaid.com/2019/02/16/stop-mentoring-first-time-contributors/) here. Up to date documentation includes (but is not limited to) docs on:
+- Installing & running the project
+- Running tests
+- Code style guidelines & tools used by your project
+- Testing requirements for new code, code coverage tools, CI, etc.
+- PR templates
+
+Another potential replacement for making/marking "beginner friendly" issues is to consider looking at larger/more complex issues and seeing how they can be broken into smaller parts. Development sprints are a GREAT time to get multiple eyes looking at those smaller parts in parallel. Also, consider reaching out to your current active contributor base (more on that below) and ask them to help with this task. They may have unique insight on good candidates for this approach.
```
Comment (drewbrew) [inline _posts/2022-07-08-dev-sprint-leaders.md]:
```suggestion
Another potential replacement for making/marking "beginner-friendly" issues is to consider looking at larger/more complex issues and seeing how they can be broken into smaller parts. Development sprints are a GREAT time to get multiple eyes looking at those smaller parts in parallel. Also, consider reaching out to your current active contributor base (more on that below) and ask them to help with this task. They may have unique insight on good candidates for this approach.
```

Same hyphen here

**Review by drewbrew — APPROVED** [review]
Thank you for writing this!

I'd love to have another review from someone with better English skills before this gets merged.

File: _posts/2022-07-08-dev-sprint-leaders.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+- It will make it easier to get new contributors started AT the sprint and going forward. This should *help* the process of getting new contributors started scale better, easing some of the individual burden on maintainers.
```
Comment (logankilpatrick) [inline _posts/2022-07-08-dev-sprint-leaders.md]:
```suggestion
- It will make it easier to get new contributors started at the sprint and going forward. This should *help* the process of getting new contributors started scale better, easing some of the individual burden on maintainers.
```

Comment (logankilpatrick) [reply _posts/2022-07-08-dev-sprint-leaders.md]: Not sure if intention or not

File: _posts/2022-07-08-dev-sprint-leaders.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+- Be prepared to take feedback from new contributors who may have trouble finding or using the New Contributor docs. If there are problems, try to help new contributors handle them and be prepared to update the appropriate documentation. That will be one less thing for the maintainers to do, and it will improve your understanding of the project.
+- Help the mainters by looking at larger/more complex issues and how they can be deconstructed into smaller tasks to spread among the newer contributors.
+- If you're already working on an issue, prepare to guide someone else to work on it with you.
+
+
+## Conclusion
+Specific time/location details on the Development Sprints will be added to [the schedule](https://2022.djangocon.us/schedule/). But for now:
+- Decide if you want to help lead or organize a Development Sprint.
+- Make sure you're prepared for whichever role you take on.
+- Get ready to have a great time!
```
Comment (logankilpatrick) [inline _posts/2022-07-08-dev-sprint-leaders.md:55]: Maybe also add your email / Twitter here Kojo so folks can get in contact if they have questions

File: _posts/2022-07-08-dev-sprint-leaders.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+If you're an active contributor to a project, even if you're not an official maintainer or someone with commit access, there's still a lot you can do to help support your current maintatiners as well as new, interested contributors.
+- Be prepared to help mentor new contributors. The two most important aspects here are patience and compassion.
+- Prepare to gauge the experience level of newer contributors, then try to mentor them accordingly. Some may be experienced Python developers, but new to Django or modern web development. Some may be experienced web developers, but with another framework/language. Some may be less experienced developers overall who want to lend a hand.
+- Be prepared to take feedback from new contributors who may have trouble finding or using the New Contributor docs. If there are problems, try to help new contributors handle them and be prepared to update the appropriate documentation. That will be one less thing for the maintainers to do, and it will improve your understanding of the project.
+- Help the mainters by looking at larger/more complex issues and how they can be deconstructed into smaller tasks to spread among the newer contributors.
```
Comment (logankilpatrick) [inline _posts/2022-07-08-dev-sprint-leaders.md]:
```suggestion
- Help the maintainers by looking at larger/more complex issues and how they can be deconstructed into smaller tasks to spread among the newer contributors.
```

**Comment by logankilpatrick** [comment]
Overall this looks great!

File: _posts/2022-07-08-dev-sprint-leaders.md (hunk 5 above)
Comment (kojoidrissa) [reply _posts/2022-07-08-dev-sprint-leaders.md:55]: [83e77f9](https://github.com/djangocon/2022.djangocon.us/pull/47/commits/83e77f99f849a50f7243ed431f4121e3ae9fc45e)

File: _posts/2022-07-08-dev-sprint-leaders.md (hunk 4 above)
Comment (kojoidrissa) [reply _posts/2022-07-08-dev-sprint-leaders.md]: [83e77f9](https://github.com/djangocon/2022.djangocon.us/pull/47/commits/83e77f99f849a50f7243ed431f4121e3ae9fc45e)

File: _posts/2022-07-08-dev-sprint-leaders.md [hunk 7]
```diff
@@ -0,0 +1,55 @@
+---
+author: DjangoCon US Organizers
+category: General
+date: 2022-07-08 13:00:00
+layout: post
+title: "Lead or organize a Development Sprint at DCUS 2022!"
+---
+
+The last two days of DjangoCon US *(this year, October 20-21)* are devoted to Development Sprints. This blog post is targeted at current open source maintainers and contributors who are considering organizing or leading a Development Sprint at DCUS 2022. As such, I won't be defining "Development Sprint". That's for another blog post.
```
Comment (williln) [inline _posts/2022-07-08-dev-sprint-leaders.md]: >As such, I won't be defining "Development Sprint". That's for another blog post. 

I'd rephrase to "A more specific definition of "Development Sprint" will come in a later post.

File: _posts/2022-07-08-dev-sprint-leaders.md [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+
+## Why lead/organize a sprint at DCUS 2022?
+There are as many reasons to lead or organize a development sprint as there are projects to work on and potential organizers. Some of the key reasons to consider are:
+- Meet & work with current contributors in person!
+- Test & refine your new user & new contributor docs with live feedback. Better new contributor docs lead to better new issues, bug fixes and PRs for your project.
+- Build/improve a workflow for virtual attendees that your project can continue to use after DjangoCon US (when EVERYONE will be 'virtual').
```
Comment (williln) [inline _posts/2022-07-08-dev-sprint-leaders.md]: > Build/improve

Change to "Build (or improve)" to get better parallelism with the first two bullets

File: _posts/2022-07-08-dev-sprint-leaders.md [hunk 9]
```diff
@@ -0,0 +1,55 @@
...
+
+A bit of preparation will help you and your chosen project get the most from the Development Sprints, as well as help potential sprinters have the best possible experience working on your project. Instead of the traditional approach of making/marking "beginner-friendly" issues, try making sure your "Start Contributing" docs are comprehensive and up to date. If your project doesn't have a CONTRIBUTING.md/rst doc, there are several examples you can look to for help, such as the two different [DRF](https://github.com/encode/django-rest-framework/blob/master/CONTRIBUTING.md) [docs](https://www.django-rest-framework.org/community/contributing/) or the [pytest](https://github.com/pytest-dev/pytest/blob/main/CONTRIBUTING.rst) docs. You can also look at the [Django](https://docs.djangoproject.com/en/dev/internals/contributing/) or [CPython](https://devguide.python.org/) developer docs. Creating or updating these docs prior to the DCUS Dev Sprints will have two main benefits:
```
Comment (williln) [inline _posts/2022-07-08-dev-sprint-leaders.md]: I'm not sure people will realize that DRF and docs link to separate places so maybe find a way to make that more clear? Maybe

> ..., there are several examples to you can to for help. Django REST Framework's [contributing guide on GitHub]() and ]on the DRF website]() are both great examples.

File: _posts/2022-07-08-dev-sprint-leaders.md [hunk 10]
```diff
@@ -0,0 +1,55 @@
...
+- It will make it easier to get new contributors started **at** the sprint *and* going forward. This should *help* the process of getting new contributors started scale better, easing some of the individual burden on maintainers.
```
Comment (williln) [inline _posts/2022-07-08-dev-sprint-leaders.md]: Fewer bolds and italics -- using both too frequently reduces overall emphasis

**Comment by williln** [comment]
Left a couple comments but LGTM!

File: _posts/2022-07-08-dev-sprint-leaders.md (hunk 8 above)
Comment (kojoidrissa) [reply _posts/2022-07-08-dev-sprint-leaders.md]: Oh! I like better parallelism!

File: _posts/2022-07-08-dev-sprint-leaders.md (hunk 9 above)
Comment (kojoidrissa) [reply _posts/2022-07-08-dev-sprint-leaders.md]: Yeah, I was iffy on that bit myself.
